dropwizard-json-logging/src/main/java/io/dropwizard/logging/json/EventJsonLayoutBaseFactory.java

```diff
@@ -0,0 +1,183 @@
+package io.dropwizard.logging.json;
+
+import ch.qos.logback.classic.LoggerContext;
+import ch.qos.logback.classic.pattern.RootCauseFirstThrowableProxyConverter;
+import ch.qos.logback.classic.pattern.ThrowableHandlingConverter;
+import ch.qos.logback.classic.spi.ILoggingEvent;
+import ch.qos.logback.core.LayoutBase;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+import io.dropwizard.logging.json.layout.EventJsonLayout;
+
+import java.util.TimeZone;
+
+/**
+ * <table>
+ * <tr>
+ * <th>Name</th>
+ * <th>Default</th>
+ * <th>Description</th>
+ * </tr>
+ * <tr>
+ * <td >{@code includeLevel}</td>
+ * <td >true</td>
+ * <td>Whether to include the logging level.</code></td>
+ * </tr>
+ * <tr>
+ * <td>{@code includeThreadName}</td>
+ * <td>true</td>
+ * <td>Whether to include the thread name.</td>
+ * </tr>
+ * <tr>
+ * <td>{@code includeMdc}</td>
```

Comment:
This is `includeMDC` (final two chars capitalized) below.

Reply:
Done